app/src/main/java/ch/hikemate/app/ui/components/HikeCard.kt

```diff
@@ -32,6 +32,8 @@ import androidx.compose.ui.unit.dp
 import ch.hikemate.app.R
 import ch.hikemate.app.ui.map.MapScreen
 
+const val TEST_TAG_HIKE_CARD_TITLE = "HikeCardTitle"
```

Comment:
Warning: you should put this into an object so that it is not available project wide as `TEST_TAG_HIKE_CARD_TITLE` but as `HikeCard.TEST_TAG_HIKE_CARD_TITLE`